edge-util/src/Microsoft.Azure.Devices.Edge.Util/json/JsonExtensions.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util.Json
+{
+    using Newtonsoft.Json;
+
+    public static class JsonExtensions
+    {
+        public static string ToJson(this object obj) => JsonConvert.SerializeObject(obj, Formatting.None);
```

Comment:
nit: just have ToJson that has Formatting as optional parameter with None by default

Reply:
Should it be more readable to have 2 different method names?